src/Bicep.Core/CoreResources.resx

```diff
@@ -156,18 +156,18 @@
     <comment>Text for linter rule error. {0} is a programmatic expression</comment>
   </data>
   <data name="ParameterMustBeUsedRuleDescription" xml:space="preserve">
-    <value>No unused parameters.</value>
+    <value>All parameters must be used.</value>
   </data>
   <data name="ParameterMustBeUsedRuleMessageFormat" xml:space="preserve">
     <value>Parameter "{0}" is declared but never used.</value>
     <comment>{0} is the paramenter name</comment>
   </data>
   <data name="PossibleSecretMessageFunction" xml:space="preserve">
-    <value>function '{0}'</value>
+    <value>Function '{0}'</value>

```

Comment:
This appears in the middle of a sentence similar to this:

This may contain possible secrets or whatever the error message was. Found: function 'listKeys()'

Reply:
```suggestion
    <value>function '{0}'</value>

```